bundles/io/org.eclipse.smarthome.io.rest.core/META-INF/MANIFEST.MF

```diff
@@ -17,13 +17,15 @@ Import-Package:
  com.google.common.base,
  com.google.common.collect,
  com.google.gson,
+ com.google.gson.reflect;version="2.7.0",
  io.swagger.annotations;resolution:=optional,
  javax.annotation.security;resolution:=optional,
  javax.ws.rs,
  javax.ws.rs.core,
  javax.ws.rs.ext,
  org.apache.commons.lang,
  org.eclipse.jdt.annotation;resolution:=optional,
+ org.eclipse.smarthome.binding.hue.handler,

```

Comment:
After spending too much time with the error message, I believe this introduces a circular dependency between the OSGi package, which fails the feature verification because neither bundle can be installed without the other.

The Resource with the new Route now requires the org.eclipse.smarthome.binding.hue.handler package, but its package (org.eclipse.smarthome.io.rest.core) is part of the esh-base feature, which is required by pretty much every other feature, including the one containing the hue binding.

To fix the problem, I suggest creating a new Resource, either in the hue binding or in a new bundle altogether, that only provides the one route for forcibly adding a discovery result.
That would also fix the layering violation this changeset introduces in the resource, where before it handled all discovery services generically but now also knows of a specific discovery service and its specific additional capability, providing a route only for it.

An alternative way to go about this could be introducing another interface that extends DiscoveryService and adds a method for forcibly adding discovery results of some kind. The new route could include a binding id to help find the correct discovery service, then check that it implements the added interface, then use that to add the given manual “discovery result”.